try
{
XYZ
}
Catch
{
$error[0].Exception
}
Finally
{
$error[0].Exception
}
